python/aie/compiler/aiecc/main.py

```diff
@@ -547,10 +547,12 @@ def run(mlir_module, args=None):
     if(opts.verbose):
         sys.stderr.write('\ncompiling %s\n' % opts.filename)
 
-    if(opts.tmpdir):
+    if opts.tmpdir:
       tmpdirname = opts.tmpdir
-    else:
+    elif opts.filename:
       tmpdirname = os.path.basename(opts.filename) + ".prj"
+    else:
+      tmpdirname = 'aiecc.prj'
```

Comment:
Should this be in a temporary directory?

Reply:
yeah that's a good idea, and can be overridden anyway if needed.